admin/src/Models/Span.js

```diff
@@ -68,12 +68,11 @@ export const mergePeriods = toMergePeriods => {
 
 // Filter not deleted spans for one category, return sorted.
 export const filterCategory = (spans, category) =>
-    spans.filter(i => !i.deleted_at &&  i.type === category).sort((a, b) => a.startdate > b.startdate);
+    spans.filter(i => !i.deleted_at &&  i.type === category).sort();
```

Comment:
Hmm, are you sure this is the correct solution? I think sort uses the string representation by default when sorting, but for an object that's typically something like "[object Object]" which would mean that there would be no sorting in practice. Is this better that it was before because spans are already sorted from the backend and that's better than a broken sorting?

Reply:
`Array.prototype.sort()` sort uses string sorting by default (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort). Perhaps that is why it works because it effectively does nothing, and it is already sorted when returned by the backend? If so, it should work equally well without sorting.

Reply:
Yes, exactly what I was trying to say. But in that case we should make sure that they always are sorted from the backend.

Reply:
I think I would prefer to sort it in the client, but based on something that works in all browsers. I would be pretty surprising to future developers if this algo in the client depends on entries coming sorted from the server.

Reply:
Without checking it's probably not sorted in the backen, but comes out in the correct order as new spans would be added last in the database.
Something like `spans.filter(i => !i.deleted_at &&  i.type === category).sort((a, b) => new Date(a.startdate) - new Date(b.startdate));` should work then.

Reply:
The Span class has `start` and `end` getter/attributes that can be used?